pytest_container/runtime.py

```diff
@@ -420,9 +411,6 @@ def __str__(self) -> str:
         return self.__class__.__name__
 
 
-LOCALHOST = testinfra.host.get_host("local://")
```

Comment:
This is a public object, please wrap this in a `try: import testinfra … except` block

Reply:
right, missed this.